contents.json

```diff
@@ -858,6 +858,17 @@
       "date_added": "Mon Jan 26 15:50:58 2015 -0800",
       "suggested_by": "@dkhamsing"
     },
+    {
+	"title": "Scoop",
+	"category-ids": ["news","news-api"],
```

Comment:
Please remove news, one category is enough

Reply:
Ok